Public/Src/Cache/ContentStore/Library/Logging/CsvFileLog.cs

```diff
@@ -182,10 +194,31 @@ public CsvFileLog
         {
             Contract.Requires(schema != null);
 
-            _schema = schema.ToArray();
-            Service = CsvEscape(serviceName ?? string.Empty);
-            Machine = CsvEscape(Environment.MachineName);
             BuildId = buildId == default ? Guid.NewGuid() : buildId;
+
+            _constColumns = new Dictionary<ColumnKind, string>
+            {
+                [ColumnKind.Empty] = string.Empty,
+                [ColumnKind.BuildId] = BuildId.ToString(),
+                [ColumnKind.Machine] = CsvEscape(Environment.MachineName),
+                [ColumnKind.Service] = CsvEscape(serviceName ?? string.Empty),

```

Comment:
shouldnt csvexcape be done at the time of serializing? Not in the internal structure.

And if here, why not osversion data?

Reply:
The assumption is that those will not contain any double quotes, which is the only char that needs escaping. 

Nevertheless, it's a good idea to just call CsvEscape and not think about it.  I'll change.  

As for whether to do it here or every time Render is called, I thought it makes sense to just do it once here.

Reply:
Changed to now escape everything when rendering.